guides/components.md

```diff
@@ -134,96 +96,113 @@ the following configuration will match the article paragraph element
 const page = create({
   scope: '.article',
 
-  textBody: text('p'),
+  textBody: {
+    scope: 'p'
+  }
 });
 
-andThen(function() {
-  assert.equal(page.textBody, 'Lorem ipsum dolor.');
-});
+assert.equal(page.textBody.text, 'Lorem ipsum dolor.');
 ```
 
-The attribute's selector can be omited when the scope matches the element we want to use.
+When `page.textBody.text` is evaluated, each link in the hierarchy of page objects has its scope incorporated into the final selector thats used to find an element whose text is returned (`.article p` in this example).
```

Comment:
When `page.textBody.text` is evaluated, each link in the hierarchy of page objects has its scope incorporated into the final selector that[']s used to find an element whose text is returned (`.article p` in this example).